source/fundamentals/data-formats/serialization.txt

```diff
@@ -0,0 +1,127 @@
+.. _csharp-serialization:
+
+=============
+Serialization
+=============
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to use the {+driver-long+} to perform
+:wikipedia:`serialization </serialization>`.
+Serialization is the process of mapping a C# object to a BSON document for
+storage in MongoDB.
+
+Serializers
+-----------
+
+Serializers are classes that handle the translation of C# objects to and
+from BSON documents. Most serializers implement the ``IBsonSerializer``
+interface. The {+driver-short+} has many built-in serializers made to handle
+primitive types, collection types, and custom classes.
+
+For a full list of available serializers, see the
+`Serializers namespace API documentation <{+api-root+}/N_MongoDB_Bson_Serialization_Serializers.htm>`__.
+
+Serializer Registry
+-------------------
+
+The serializer registry is an implementation of the ``IBsonSerializerRegistry`` interface
+that contains all registered serializers. Before you can use a serializer, you
+must register it by adding it to the serializer registry, as
+shown in the following example:
+
+.. code-block:: csharp
+
+   BsonSerializer.RegisterSerializer(typeof(MySerializerType), new MySerializer); 
+
+You can access the serializer registry with the ``SerializerRegistry`` property
+of the ``BsonSerializer`` class as follows:
+
+.. code-block:: csharp
+
+   var intSerializer = BsonSerializer.SerializerRegistry.GetSerializer<int>();
+
+.. important::
+
+   The serializer registry is a global registry. This means that you cannot use
+   multiple registries in a single application.
+
+Custom Serializers
+~~~~~~~~~~~~~~~~~~
+
+In some cases, you might need to create a custom serializer. When creating a
```

Comment:
S: Does "in some cases" mean "if you need to serialize a custom type"? If so, probably clearer to just say that.